nf_core/pipelines/bump_version.py

```diff
@@ -101,7 +101,7 @@ def bump_pipeline_version(pipeline_obj: Pipeline, new_version: str) -> None:
     )
     # nf-test snap files
     pipeline_name = pipeline_obj.nf_config.get("manifest.name", "").strip(" '\"")
-    snap_files = [f for f in Path().glob("tests/pipeline/*.snap")]
+    snap_files = [f for f in Path(pipeline_obj.wf_path).glob("tests/pipeline/*.snap")]
```

Comment:
I think the workflow path is not needed, we use relative paths for all other paths in `update_file_version `, so we should modify the file paths resulting from this `glob`

Reply:
The tests don't pass otherwise because it is looking for files inside the tools repo. Or am I not setting the tests up correctly?

Reply:
I see it is working, I am curious why it does also work for the other files, maybe we should add `pipeline_obj.wf_path` to all of them

Reply:
actually, I was doing more tests, and when I run the command `nf-core pipelines bump-version -d nf-core-test`, the other files work but this one shows a warning
`WARNING  File not found: 'nf-core-test/nf-core-test/tests/pipeline/main.nf.test.snap'`

I think we need to pass the workflow directory to this `glob()` funciton, but strip it from the final paths.

Reply:
sorry, what do you mean with final paths?

Reply:
I think this should make it
```suggestion
    snap_files = [f.relative_to(pipeline_obj.wf_path) for f in Path(pipeline_obj.wf_path).glob("tests/pipeline/*.snap")]
```

Reply:
ah,  yes makes sense.